tools/tests/bundle-ignore-files.js

```diff
@@ -12,4 +12,10 @@ selftest.define("bundle-ignore-files", () => {
   ];
 
   matchingInputs.forEach(input => selftest.expectEqual(patterns.some(p => p.test(input)),true));
+
+  const nonMatchingInputs = [
+    '/imports/components/Icon/index.js',
```

Comment:
@mjmasn Wow...what the file!? That was very likely the original intention.  Thanks for finding that.

@sebakerckhof Could you update?